src/trix/config/block_attributes.coffee

```diff
@@ -5,6 +5,10 @@ Trix.config.blockAttributes = attributes =
   quote:
     tagName: "blockquote"
     nestable: true
+  heading:
```

Comment:
I can see both sides. 

Is there a downside to using `heading-1`, `heading-2`, etc for the attribute names, and using @javan's semantic names for the titles? IE, in [config/lang](https://github.com/basecamp/trix/blob/master/src/trix/config/lang.coffee):

```cs
Trix.config.lang =
  header1: "Heading"
  header2: "Subheading"
```